.travis.yml

```diff
@@ -13,14 +13,14 @@ env:
 - DB=pg
 - DB=sqlite3
 gemfile:
-- gemfiles/virtual_attributes_50.gemfile
-- gemfiles/virtual_attributes_51.gemfile
-#- gemfiles/virtual_attributes_52.gemfile
-#- gemfiles/virtual_attributes_60.gemfile
+- gemfiles/gemfile_50.gemfile
```

Comment:
No where to put this, but there is a typo in your commit message:

> So it is easier ff they all are basically the same with the same file
> names.

`s/ff/if/`